src/main/java/net/kodehawa/mantarobot/commands/UtilsCmds.java

```diff
@@ -291,64 +289,103 @@ private static List<ReminderObject> getReminders(List<String> reminders) {
-                                "https://www.mantaro.site/mantaro-wiki/currency/101"))
-                .addSubCommand("items", (ctx, s) ->
-                        ctx.send(EmoteReference.OK + "**For a list of items, please visit:**" +
-                                "https://www.mantaro.site/mantaro-wiki/currency/items"))
-                .addSubCommand("overview", (ctx, s) ->
-                        ctx.send(EmoteReference.OK + "**For a feature overview, check:** https://mantaro.site/features.html"))
-                .addSubCommand("birthday", (ctx, s) ->
-                        ctx.send(EmoteReference.OK + "**For a guide on the birthday system, please visit:**" +
-                                "https://www.mantaro.site/mantaro-wiki/guides/birthday-announcer"))
-        );
+            protected void process(TextContext ctx) {
+                ctx.send(EmoteReference.OK + "**For Mantaro's ToS please visit:** https://www.mantaro.site/mantaro-wiki/legal/terms-of-service");
+            }
+        }
+
+        public static class UserMessage extends TextCommand {
+            @Override
+            protected void process(TextContext ctx) {
+                ctx.send(EmoteReference.OK + "**For Mantaro's Welcome and Leave message tutorial please visit:** https://www.mantaro.site/mantaro-wiki/guides/welcome-and-leave-messages");
+            }
+        }
+
+        public static class Premium extends TextCommand {
+            @Override
+            protected void process(TextContext ctx) {
+                ctx.send(EmoteReference.OK + "**To see what Mantaro's Premium features offer please visit:** https://www.mantaro.site/mantaro-wiki/basics/premium-perks");
+            }
+        }
+
+        public static class Currency extends TextCommand {
+            @Override
+            protected void process(TextContext ctx) {
+                ctx.send(EmoteReference.OK + "**For a Currency guide, please visit:** https://www.mantaro.site/mantaro-wiki/currency/101");
+            }
+        }
+
+        public static class Items extends TextCommand {
+            @Override
+            protected void process(TextContext ctx) {
+                ctx.send(EmoteReference.OK + "**For a list of items, please visit:** https://www.mantaro.site/mantaro-wiki/currency/items");
+            }
+        }
+
+        public static class Overview extends TextCommand {
+            @Override
+            protected void process(TextContext ctx) {
+                ctx.send(EmoteReference.OK + "**For a feature overview, check:** https://mantaro.site/features.html");
+            }
+        }
+
+        public static class Birthday extends TextCommand {
+            @Override
+            protected void process(TextContext ctx) {
+                ctx.send(EmoteReference.OK + "**For a guide on the birthday system, please visit:** https://www.mantaro.site/mantaro-wiki/guides/birthday-announcer");
+            }
+        }
```

Comment:
Realistically do all of these need to be separate classes? We could just have the argument resolved in a single class. Might need some hackery for help context tho.